CONTRIBUTING.md

```diff
@@ -26,5 +26,7 @@ Follow either of the two links above to access the appropriate CLA and instructi
 1. Submit an issue describing your proposed change to the repo in question.
 1. The repo owner will respond to your issue promptly.
 1. Fork the desired repo, develop and test your code changes.
+1. In the directory for each of the packages that you have made changes in, run `npm run test` to run the tests and check coverage. Ensure there are no new failures after your changes. 
+1. From the top level directory run `npm run check` and address any failures. This builds all of the packages and runs the linter. This ensures changes you have made do not affect any of the packages (which can happen even if you have not changed them specifically).
```

Comment:
Optional: "Use `npm run fix` to fix linter failures." Can we add this line also?

Reply:
@mayurkale22 done, although I used slightly different wording.